src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertSolutions.sql

```diff
@@ -282,6 +282,36 @@ BEGIN
 
     /*************************************************************************************************************************************************************/
 
+    SET @solutionId = '100008-001';
```

Comment:
As the first part of the solution ID is the supplier ID, this solution ID should be `'100007-002'`.